corehq/util/context_processors.py

```diff
@@ -114,11 +114,14 @@ def current_url_name(request):
 def js_api_keys(request):
     if hasattr(request, 'couch_user') and request.couch_user and not request.couch_user.analytics_enabled:
         return {}  # disable js analytics
-    return {
+    api_keys = {
         'ANALYTICS_IDS': settings.ANALYTICS_IDS,
         'ANALYTICS_CONFIG': settings.ANALYTICS_CONFIG,
         'MAPBOX_ACCESS_TOKEN': settings.MAPBOX_ACCESS_TOKEN,
     }
+    if hasattr(request, 'project') and request.project.hipaa_compliant:
```

Comment:
To be conservative, should this also check that `GOOGLE_ANALYTICS_API_ID` is in `api_keys['ANALYTICS_IDS']`? Non-Dimagi environments may not have it.

Reply:
We set that in `settings.py` to an empty string so someone would have to intentionally go in and override that locally for it to be missing

Reply:
So it'll be missing if they set up other analytics but not GA, right? (Yes, this is a narrow edge case.)

Reply:
yeah, that does sound right. if the replace the dict with one that does not include all keys, this will break. I can fix it, should be easy enough

Reply:
thanks